templates/Default/footer.html.twig

```diff
@@ -69,8 +69,8 @@
         </select>
       </div>
       <div class="copyright footer-right">
-        Copyright &copy; {{ "now"|date("Y") }} <a href="http://www.catrobat.org">Catrobat</a>. All
-        rights reserved.
+        {{ "copyright" | trans({}, "catroweb") }} &copy;
```

Comment:
I think it would be better to use a single translation with the year as variable.

Reply:
But then we have to delete the Translation every year..

Reply:
If you use the year as variable/parameter/argument, no.
See second point of commit message.

Reply:
Point taken :)